pkg/chunk/encoding/factory.go

```diff
@@ -52,6 +52,8 @@ const (
 	Varbit
 	// Bigchunk encoding
 	Bigchunk
+	// PrometheusXorChunk is a wrapper around Prometheus XOR-encoded chunk.
+	PrometheusXorChunk
```

Comment:
Why not just calling it `XORChunk` (as named in Prometheus)?

Reply:
I wanted to make it clear that this is using XorChunk from Prometheus, hence the name.